src/features/sponsor-dashboard/components/Submissions/SubmissionDetails.tsx

```diff
@@ -171,7 +172,7 @@ export const SubmissionDetails = ({
   }) => {
     setIsPaying(true);
     try {
-      let transaction;
+      const transaction = new Transaction();
```

Comment:
my changes from a different PR are showing up in this PR's diff
so i would suggest you to open a separate PR by checking out from the latest staging branch